tests/test_trainable_tokens.py

```diff
@@ -978,3 +978,119 @@ def test_scaled_embedding_with_lora(self):
             orig_embedding.embed_scale.fill_(0)
             embedding_output = peft_embedding(x)
             assert (embedding_output == 0.0).all()
+
+    # Tests for ensure_weight_tying parameter with trainable_token_indices
```

Comment:
Let's mention https://github.com/huggingface/peft/issues/2864 here, I think it helps understanding the tests.